transifex/pull.py

```diff
@@ -218,11 +226,16 @@ def pull(repo):
 
 
 if __name__ == '__main__':
-    settings_file = join(abspath(dirname(__file__)), 'settings.yaml')
-    with open(settings_file) as f:
-        settings = yaml.load(f)
-
-    with concurrent.futures.ProcessPoolExecutor() as executor:
-        for clone_url in settings['repos']:
-            repo = Repo(clone_url)
-            executor.submit(pull, repo)
+    try:
+        clone_url = sys.argv[1]
+        repo = Repo(clone_url)
+        pull(repo)
+    except IndexError:
```

Comment:
I guess it's a stylistic choice. This is what exception handlers are meant for. You could achieve the same effect with `len()` and a conditional, too. I think what I have here is pretty explicit.